src/apps/api/serializers/leaderboards.py

```diff
@@ -93,7 +95,7 @@ def get_submissions(self, instance):
         # asc == colname
         primary_col = instance.columns.get(index=instance.primary_index)
         ordering = [f'{"-" if primary_col.sorting == "desc" else ""}primary_col']
-        submissions = instance.submissions.all().select_related('owner').prefetch_related('scores').annotate(primary_col=Sum('scores__score', filter=Q(scores__column=primary_col)))
+        submissions = Submission.objects.filter(leaderboard=instance).select_related('owner').prefetch_related('scores').annotate(primary_col=Sum('scores__score', filter=Q(scores__column=primary_col)))
```

Comment:
could we split this guy into multiple lines + add a comment? just in case we need to look at this 2 years from now, will speed up grokkin!